src/overlays/actors/ovl_En_In/z_en_in.c

```diff
@@ -33,75 +36,938 @@ const ActorInit En_In_InitVars = {
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_In/func_80A7AEF0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_In/func_80A7B018.s")
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK10, 0x00, 0x00, 0x39, 0x20, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+    { 18, 46, 0, { 0, 0, 0 } },
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_In/func_80A7B024.s")
+static CollisionCheckInfoInit2 sColChkInfoInit = {
+    0x00, 0x0000, 0x0000, 0x0000, 0xFF,
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_In/EnIn_Update.s")
+static struct_D_80AA1678 sAnimationInfo[] = {
+    { 0x06001CC0, 1.0f, 0, 0.0f },   { 0x06001CC0, 1.0f, 0, -10.0f }, { 0x06013C6C, 1.0f, 0, 0.0f },
+    { 0x06013C6C, 1.0f, 0, -10.0f }, { 0x06000CB0, 1.0f, 0, 0.0f },   { 0x060003B4, 1.0f, 0, -10.0f },
+    { 0x06001BE0, 1.0f, 0, 0.0f },   { 0x06013D60, 1.0f, 0, 0.0f },   { 0x0601431C, 1.0f, 0, 0.0f },
+    { 0x06014CA8, 1.0f, 0, 0.0f },
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_In/func_80A7B320.s")
+static AnimationHeader* D_80A7B918[] = {
+    0x060151C8, 0x06015DF0, 0x06016B3C, 0x06015814, 0x0601646C, 0x060175D0, 0x06017B58, 0x06018C38,
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_In/func_80A7B570.s")
+static Gfx* sAlternateOutfitDLs[] = {
+    NULL,       NULL,       0x06013688, 0x060137C0, 0x06013910, 0x060132B8, 0x060133F0,
+    0x06013540, 0x06013100, 0x06011758, 0x06012AC0, 0x06012BF0, 0x06012D20, 0x06012480,
+    0x060125B0, 0x060126E0, 0x06011AD8, 0x06012350, 0x06012220, 0x06012120,
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_In/EnIn_Draw.s")
+extern AnimationHeader D_060003B4;
+extern AnimationHeader D_06001BE0;
+extern AnimationHeader D_06013D60;
+extern AnimationHeader D_06014CA8;
+extern AnimationHeader D_06015814;
+extern AnimationHeader D_0601646C;
+extern AnimationHeader D_06018C38;
+extern FlexSkeletonHeader D_06013B88;
+extern UNK_TYPE D_060034D0;
+extern Gfx D_06007A20[];
+extern Gfx D_06007BF8[];
+
+u16 func_80A78FB0(GlobalContext* globalCtx) {
+    if (gSaveContext.eventChkInf[1] & 0x10) {
+        if (gSaveContext.infTable[9] & 0x80) {
+            return 0x2046;
+        } else {
+            return 0x2045;
+        }
+    }
+    if (gSaveContext.infTable[9] & 0x10) {
+        return 0x2040;
+    } else {
+        return 0x203F;
+    }
+}
+
+u16 func_80A79010(GlobalContext* globalCtx) {
+    Player* player = PLAYER;
+    u16 temp_v0 = Text_GetFaceReaction(globalCtx, 25);
+
+    if (temp_v0 != 0) {
+        return temp_v0;
+    }
+    if (gSaveContext.eventChkInf[1] & 0x100) {
+        if (gSaveContext.nightFlag == 0) {
+            return 0x205F;
+        } else {
+            return 0x2057;
+        }
+    }
+    if (gSaveContext.nightFlag == 1) {
+        return 0x204E;
```

Comment:
if night

Reply:
This doesn't match since it's a direct comparison to 1 as opposed to != 0